exercises/transpose/src/main/java/Transpose.java

```diff
@@ -0,0 +1,5 @@
+public class Transpose {
```

Comment:
According to the [POLICIES](https://github.com/exercism/java/blob/master/POLICIES.md) doc, since this exercise is now after the first 20 exercises, it shouldn't have any starter implementation. Instead there should just be a `.keep` file in this directory :)